packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/layout.tsx

```diff
@@ -2,12 +2,15 @@
 
 import { Navigation } from "../../../../../components/Navigation";
 import { Providers } from "./Providers";
+import { TransactionsWatcher } from "./observe/TransactionsWatcher";
 
 export default function WorldLayout({ children }: { children: React.ReactNode }) {
   return (
     <Providers>
-      <Navigation />
-      {children}
+      <TransactionsWatcher>
```

Comment:
this doesn't seem to do anything as a component, just does hook-like things, so wondering if this can just be

```
useTransactionsWatcher()
```
or similar at the top of this component?

also wondering if we need to include this in some higher-up component to capture messages across all pages, not just a given world?

Reply:
ohh I thought this was the observer listener

re: component, could just have it return `null` then you can have it as a sibling instead of a wrapper here

Reply:
Good idea, doesn't have to be a parent indeed.

Also added `observerStore` "initializer" in `WorldsForm.tsx` with `useStore(observerStore)` in order to start observing for local events higher-up the tree as discussed. Not a huge fan of doing it this way but only importing it didn't work, and setting up some kind of `init` function just adds boilerplate. Curious what you think.